package.json

```diff
@@ -51,6 +51,7 @@
     "react-dom": "^16.13.1",
     "react-scripts": "^3.4.1",
     "reakit": "^1.1.2",
+    "reakit-utils": "^0.13.1",
```

Comment:
I don't know if the correct is `devDependencies` or `peerDependencies`, because I only use the types of `reakit-utils` here